static/src/app/library/library.js

```diff
@@ -0,0 +1,20 @@
+import angular from 'angular';
+import leaflet from 'leaflet';
```

Comment:
Yeah I know that. I was just confused because I saw it and wasn't sure if you meant to have it working or not.

As per our conversation though we can just delete all this stuff so we can add it in in a piecemeal, working state so don't worry about getting it set up now.